packages/api/src/middleware/checkAnimalEntities.js

```diff
@@ -189,6 +190,25 @@ export function validateAnimalBatchCreationBody(animalBatchKey) {
           }
         }
 
+        const relationshipsKey =
+          animalBatchKey === 'batch'
+            ? 'animal_batch_use_relationships'
+            : 'animal_use_relationships';
+
+        if (animalOrBatch[relationshipsKey]) {
+          if (!Array.isArray(animalOrBatch[relationshipsKey])) {
+            return res.status(400).send(`${relationshipsKey} must be an array`);
+          }
+
+          const otherUse = await AnimalUseModel.query().where({ key: 'OTHER' }).first();
+
+          for (const relationship of animalOrBatch[relationshipsKey]) {
+            if (relationship.use_id != otherUse.id && relationship.use_other) {
```

Comment:
This is meant to match the field name, right?

```suggestion
            if (relationship.use_id != otherUse.id && relationship.other_use) {
```

Reply:
Thanks!